invenio_app_rdm/users_ui/templates/semantic-ui/invenio_app_rdm/users/header.html

```diff
@@ -23,14 +23,14 @@ <h1 class="ui medium header" aria-label="{{ _('Your dashboard') }}">{{ current_u
       </div>
     </div>
     {% if active_dashboard_menu_item == "uploads" %}
-      <a class="ui tiny button positive left labeled icon m-0" href="/uploads/new">
+      <a class="ui tiny button positive left labeled icon m-0" href={{ url_for("deposit_create") }}>
```

Comment:
@sakshamarora1 this should be actually `invenio_app_rdm_records.deposit_create` (since the blueprint has [this prefix](https://github.com/inveniosoftware/invenio-app-rdm/blob/a4a7240114825386de6a9ec02d5febdfc92d3014/invenio_app_rdm/records_ui/views/__init__.py#L71)). This is also relevant to my comment about being explicit about the endpoint naming.

Reply:
is this working in flask/werkzeug <3.0.0 because with flask,werkzeug >3.0 it fails without the `invenio_app_rdm_records` prefix